<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>961</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenImg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 215, 215); border-color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string>Img</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>DFT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 215, 215); border-color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string>spectrum</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 215, 215); border-color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string>phase angle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 215, 215); border-color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string>IDFT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenImg</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_filter">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>640</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ideal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Butterworth</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gaussian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Homomorphic</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>580</y>
      <width>320</width>
      <height>320</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 215, 215); border-color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string>Img</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_apply">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>800</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_restore">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>860</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Restore</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>670</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Pass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High pass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Low pass</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_yH">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>750</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="singleStep">
     <double>0.200000000000000</double>
    </property>
    <property name="value">
     <double>3.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_yL">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>750</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="singleStep">
     <double>0.200000000000000</double>
    </property>
    <property name="value">
     <double>0.400000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_yH">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>780</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>yH</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_yL">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>780</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>yL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>590</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenImg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>580</y>
      <width>320</width>
      <height>320</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 215, 215); border-color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string>Img</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>790</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inverse</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wiener</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_motion_blurred">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>650</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Motion blurred</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reconstruct">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>850</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reconstruct</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>820</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussain noises</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_cutoff_frequency">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>700</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_cutoff_frequency">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>730</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>D0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_idft">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>IDFT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_difference">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Difference</string>
    </property>
   </widget>
   <widget class="QLabel" name="label7">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>70</y>
      <width>480</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 215, 215); border-color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string>DFT vs IDFT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HW4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palace Script MT</family>
      <pointsize>26</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>TA so handsome</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1266</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
